### PR TITLE
Non dockerized build

### PR DESCRIPTION
**Pull request by yuvalif**
this is a blog post regarding non-dockerized build

### CODE REVIEW

File: _posts/2018-06-07-Non-Dockerized-Build.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+### Targets
+
+To execute any of the targets use:
+```
+make -f Makefile.nocontainer <target>
+```
+File has the following targets:
+
+ - **bootsrap**: this is actually part of the prerequisites, but added all golang tool dependencies here, since this is agnostic of the running platform Should be called once
```
Comment (karmab) [inline _posts/2018-06-07-Non-Dockerized-Build.markdown]: typo bootstrap

File: _posts/2018-06-07-Non-Dockerized-Build.markdown [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+# Why?
+
+Current build system for KubeVirt is done inside docker. This ensures a robust and consistent build environment:
+- No need to install system dependencies
+- Controlled versions of these dependencies
+- Agnostic of local golang environment
+
+So, in general, **you should just use the dockerized build system**.
+
+Still, there are some drawback there:
```
Comment (karmab) [inline _posts/2018-06-07-Non-Dockerized-Build.markdown]: drawbacks

File: assets/2018-06-07-Non-Dockerized-Build/Makefile.nocontainer [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	./hack/functests.sh
+
+cluster-sync: build
+	./hack/build-copy-artifacts.sh
+	./hack/build-manifests.sh
+	./hack/build-docker.sh build
+	./cluster/clean.sh
+	./cluster/deploy.sh
+
+.PHONY: bootstarp generate apidocs build test check functest cluster-sync
```
Comment (karmab) [inline assets/2018-06-07-Non-Dockerized-Build/Makefile.nocontainer]: bootstrap

File: _posts/2018-06-07-Non-Dockerized-Build.markdown [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+*Similarly to the containerized case*, docker is still needed (e.g. all the cluster stuff is done via docker),  and therefore, any docker related preparations are needed as well. This would include running docker on startup and making sure that docker commands does not need root privileges. On Fedora27 this would mean:
+```
+sudo groupadd docker
+sudo usermod -aG docker $USER
+sudo systemctl enable docker
+sudo systemctl start docker
+```
+Now, getting the actual code could be done either via `go get` (don't forget to set the `GOPATH` environment variable):
+```
+go get -d -insecure kubevirt.io/kubevirt/...
```
Comment (rmohr) [inline _posts/2018-06-07-Non-Dockerized-Build.markdown]: The `-insecure` can now be dropped. You should also not need the `/...` if I am not mistaken.

**Comment by scollier** [comment]
LGTM.

**Pull request merged**